### PR TITLE
basic react framework

### PR DESCRIPTION
**Pull request by jessikka12**
without the annotations

### CODE REVIEW

File: src/Annotation.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
```
Comment (charlesma11) [inline src/Annotation.js:1]: Lets put these files into a folder called `components`

File: src/AnnotationList.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <Divider />
+        <Annotation 
+          title="Second Annotation" 
+          description={"maybe we can fill these in by reading from a spreadsheet?"}
+        />
+        <Divider />
+        <Annotation
+          title="Videos?"
+          description={"for videos, we can pass in a boolean to the annotation for if theres a video and display a video if the boolean is true"}
+          embedId='vvN4FgqNXwE'
```
Comment (charlesma11) [inline src/AnnotationList.js:45]: what is this?

File: src/App.js [hunk 3]
```diff
@@ -1,19 +1,66 @@
...
 function App() {
-  const { unityProvider } = useUnityContext({
+  const { unityProvider, requestPointerLock } = useUnityContext({
     loaderUrl: 'build/webIntegration.loader.js',
     dataUrl: 'build/webIntegration.data',
     frameworkUrl: 'build/webIntegration.framework.js',
     codeUrl: 'build/webIntegration.wasm',
   });
 
+  const [tutOpen, setTutOpen] = useState(true);
```
Comment (charlesma11) [inline src/App.js:17]: lets rename these states to something like `tutorialIsOpen`

Comment (charlesma11) [reply src/App.js:17]: length is not as important as good naming

File: src/Tutorial.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogActions from '@mui/material/DialogActions';
+import Button from '@mui/material/Button';
+import DialogContentText from '@mui/material/DialogContentText';
+
+export default function Tutorial({open, onClose}) {
+
+    return (
+        <Dialog
```
Comment (charlesma11) [inline src/Tutorial.js:12]: Reason for using Dialog here over Typography?

File: src/YoutubEmbed.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const YoutubeEmbed = ({ embedId }) => (
```
Comment (charlesma11) [inline src/YoutubEmbed.js:4]: we can stick with this for now, will just need Lauren to upload vids as youtube links

File: src/App.css [hunk 6]
```diff
@@ -36,3 +36,25 @@
     transform: rotate(360deg);
   }
 }
+
+#title-bar {
+  height: 7vh;
```
Comment (charlesma11) [inline src/App.css:41]: y vh over px?

**Review by charlesma11 — CHANGES_REQUESTED** [review]

File: src/Annotation.js (hunk 1 above)
Comment (charlesma11) [reply src/Annotation.js:1]: so the `Annotation.js`, `AnnotationList.js`, etc. all the components

File: src/Tutorial.js (hunk 4 above)
Comment (jessikka12) [reply src/Tutorial.js:12]: dialog is the thing that pops up over the screen. typography is just a fancy text field

File: src/App.css (hunk 6 above)
Comment (jessikka12) [reply src/App.css:41]: vh let's u do a percentage of the screens height. for different sized devices

File: src/App.js (hunk 3 above)
Comment (jessikka12) [reply src/App.js:17]: sure

File: src/Annotation.js (hunk 1 above)
Comment (jessikka12) [reply src/Annotation.js:1]: sure

File: src/AnnotationList.js (hunk 2 above)
Comment (jessikka12) [reply src/AnnotationList.js:45]: the embedId?? that's the identifier for the video, it's the part in the youtube url link after "https://www.youtube.com/watch?v="

**Review by charlesma11 — APPROVED** [review]
lgtm

**Pull request merged**